cassandra-scheduler/src/main/java/io/mesosphere/mesos/frameworks/cassandra/ApiController.java

```diff
@@ -713,4 +734,44 @@ private static Response nodeStatusUpdate(CassandraFrameworkProtos.CassandraNode
         }
         return Response.status(status).entity(sw.toString()).type("application/json").build();
     }
+
+    @GET
+    @Path("/node/replace/{node}")
+    public Response nodeReplace(@PathParam("node") String node) {
```

Comment:
Nope - the operator's responsible to replace a node. It's nothing that can or should be done automatically.

Of course, there are multiple scenarios:
* manual, planned node replacement: for example when you scale up one node after another (replacing with "better" hardware)
* determining that a node is no longer available (specific to Cassandra-on-Mesos): But when exactly is a node "no longer available"?

We should be very careful about doing such things automatically.